lib/schemes/refresh.js

```diff
@@ -69,21 +80,80 @@ export default class RefreshScheme extends LocalScheme {
       this.options.endpoints.login
     )
 
-    // Update tokens
+    /*
+    * Assign tokens based on the `property` key provided in the `auth.strategy_name.token` or `auth.strategy_name.refreshToken` config
+    * auth:{
+    *   strategy_name:{
+    *     token:{
+    *       property:'token'
+    *     },
+    *
+    *     refreshToken:{
+    *       property:'refresh'
+    *     }
+    *   }
+    * }
+    * */
+
     this.$auth.token.set(getResponseProp(response, this.options.token.property))
     this.$auth.refreshToken.set(getResponseProp(response, this.options.refreshToken.property))
 
-    // Update client id
+    /*
+    * Assign a client id from the JSON response
+    *
+    * Scheme configuration example:
+    * auth:{
+    *   strategy_name:{
+    *     clientId:{
+    *       property:'clientid'
+    *     }
+    *   }
+    * }
+    * */
     if (this.options.clientId) {
       this._setClientId(getResponseProp(response, this.options.clientId.property))
     }
 
-    // Initialize scheduled refresh if `autoRefresh` is enabled
+    /*
+    * If the Scheme's `autoRefresh` is true then it will decode the token for the expiry duration
+    * and set a `setTimeOut` callback to refresh at 75% of the expiry duration.
+    * */
```

Comment:
Can you remove this comment as auto refresh was removed?

Reply:
ok